src/dream/QueryDriver/Postgres.ts

```diff
@@ -162,21 +162,33 @@ export default class PostgresQueryDriver<
     const columnToDBTypeMap = await sqlQuery.execute(db)
     const rows = columnToDBTypeMap.rows as SchemaBuilderInformationSchemaRow[]
 
+    const dreamApp = DreamApp.getOrFail()
+
     const columnData: {
       [key: string]: SchemaBuilderColumnData
     } = {}
+
     rows.forEach(row => {
       const isEnum = ['USER-DEFINED', 'ARRAY'].includes(row.dataType) && !isPrimitiveDataType(row.udtName)
       const isArray = ['ARRAY'].includes(row.dataType)
       const associationMetadata = associationData[row.columnName]
 
+      const isPolymorphicTypeField = Object.keys(dreamApp.models).some(key => {
+        const modelClass = dreamApp.models[key]
+        if (modelClass?.table !== tableName) return false
+        if (!(modelClass?.['polymorphicTypeColumns']?.() || []).includes(camelize(row.columnName)))
+          return false
+        return true
+      })
+
       columnData[camelize(row.columnName)] = {
         dbType: row.udtName,
         allowNull: row.isNullable === 'YES',
         enumType: isEnum ? this.enumType(row) : null,
         enumValues: isEnum ? `${this.enumType(row)}Values` : null,
         isArray,
         foreignKey: associationMetadata?.foreignKey || null,
```

Comment:
Do we need to do the same thing you're doing here for `isPolymorphicForeignKeyField` since polymorphic foreign keys aren't true foreign keys and won't be picked up by this row?

Reply:
It seems not because of `DreamBelongsToForeignKeys<DreamInstance>`. 🤔 If that's able to extract foreign key from the association, could we just extract the foreign key type from the association as well and not do all this with `isPolymorphicTypeField`?